docs/_sass/minimal-mistakes/_base.scss

```diff
@@ -31,14 +31,14 @@ h6 {
   line-height: 1.2;
   font-family: $header-font-family;
   font-weight: bold;
-  color: #007edf;
+  color: #000000;
 }
 
 h1 {
   margin-top: 0;
   font-size: $type-size-3;
-  color: #ed1c24;
-}
+  color: #ff8c00;
+}ff8c00
```

Comment:
I've never written scss before, so this looks like an error to me?

Reply:
You're right.

Reply:
Pressed CTRL-C once too many